exploration/expression-attributes.md

```diff
@@ -102,7 +144,7 @@ Define the meaning and supported values of some expression attributes in the spe
 including at least `@dir` and `@locale`.
 
 To support later extension of the specified set of attributes while allowing user extensibility,
-require custom attribute names to include a U+002D Hyphen-Minus `-`.
+suggest custom attribute names to include a U+002D Hyphen-Minus `-`.
```

Comment:
This might be related to #475 ?

Reply:
Yeah, this is ultimately dependent on resolving namespacing.